README.md

```diff
@@ -18,6 +18,7 @@ This role supports Centos 7 and Ubuntu 16.04 Xenial.
 | `openio_zk_cluster_ip`		| `['127.1.1.1', '127.2.2.2', '127.3.3.3']`	| List of IP addresses where to install the Zookeeper cluster (3 IPs min / 5 IPs max) |
 | `conscience_ip`	| `'127.1.1.1'`					| IP address where to put the conscience service |
 | `openio_redis_cluster_ip`		| `['127.1.1.1', '127.2.2.2', '127.3.3.3']`					| List of IP addresses of the Redis cluster |
+| `openio_memcache_farm_ip`             | `['127.1.1.1', '127.2.2.2', '127.3.3.3']`                                     | List of IP addresses of memcached farm
```

Comment:
I'd use `openio_memcache_cluster_ip` for consistency with other ip lists

Reply:
Please, fix this and I'll accept :)